common/src/main/java/com/microsoft/identity/common/internal/logging/Logger.java

```diff
@@ -315,7 +315,8 @@ public static void info(final String tag, @Nullable final String message) {
         getInstance().log(
                 tag,
                 LogLevel.INFO,
-                DiagnosticContext.getRequestContext().toJsonString(),
+                DiagnosticContext.THREAD_ID + " : " + DiagnosticContext.getRequestContext().getOrDefault(DiagnosticContext.THREAD_ID, "UNSET") + ", " +
+                        DiagnosticContext.CORRELATION_ID + " : " + DiagnosticContext.getRequestContext().getOrDefault(DiagnosticContext.CORRELATION_ID, "UNSET"),
```

Comment:
Why making this change?
If we are making it, then why only for INFO level? Why not all levels?
If we are making it, then can this entire block of code move to its own method?

Reply:
Yes, this can be applied to all the logging levels, and would move it in its own method.
To give some context, `DiagnosticContext.getRequestContext().toJsonString()` was working initally because, the `RequestContext ` only contained the CorrelationId and ThreadId. But additional items had to be introduced for tracking the SKUs in the  `RequestContext` so soon the logs started to get spammed with additional metadata(clutter).

Reply:
made the change to all the levels :https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1237/commits/e3620b61cfa0376f89d61d646bfe46340d2f295d